sycl/doc/SYCLCompilerAndRuntimeDesign.md

```diff
@@ -217,17 +217,16 @@ and understands mnemonics designating a particular code form, for example
 architecture).  User can specify desired code format using the target-specific
 option mechanism, similar to OpenMP.
 
-`-Xsycl-target=<triple> <arg>`
+`-Xsycl-target-backend=<triple> "<arg>,<arg>"`
 
-For example, to support offload to CPU, FPGA, Gen9/vISA3.3, Gen9/SPIR-V the
-following options would be used:
+For example, to support offload to Gen9/vISA3.3, the following options would be used:
 
-`-fsycl -fsycl-targets=x86,fpga,gen9 -Xsycl-target=gen9 "-fmt:visa -fmt:spirv"`
+`-fsycl -fsycl-targets=spir64_gen-unknown-<os>-sycldevice -Xsycl-target-backend "-device skl"`
 
 The `<arg>` parameter is passed by the driver directly to the SYCL device
```

Comment:
```suggestion
The `-device skl` parameter is passed by the driver directly to the Gen device compiler backend.
```

Reply:
Thanks for the suggestions, applying soon